geode-assembly/src/distributedTest/java/org/apache/geode/management/internal/cli/commands/StartServerCommandDUnitTest.java

```diff
@@ -240,46 +241,64 @@ public void testWithAvailablePort() throws IOException {
   @Test
   public void testWithMissingStartDirectoryThatCannotBeCreated() {
```

Comment:
For these permissions tests, you could potentially abstract all file manipulation away from the class(es) under test, add a constructor that let's you pass in YourFileManipulationClass and then have the test create a YourFileManipulationClass that throws permissions exceptions.

We don't really need to test with the file system throwing exceptions at our code. We just want to verify the behavior of "start server" when it's unable to create a directory. Maybe a mocking based test would be better.

Reply:
Not sure if this is what you were thinking, but I've moved those dir creation failure tests into StartMemberUtilsTest where the result of `File.mkdirs()` can be mocked to simulate the failure.